src/js/components/PostCard/PostCard.tsx

```diff
@@ -41,6 +41,7 @@ export type PostType = {
 
 export type Props = {
   actions?: ButtonProps[];
+  addLabel?: any;
```

Comment:
```suggestion
  addLabel?: Partial<ButtonProps>;
```
Could we maybe do this for more security?

Reply:
Good point, I usually add types once I finish everything else, I just forgot to do that in the end